Mod2/Lessons/Week2/GitForPairs.md

```diff
@@ -7,75 +7,71 @@ permalink:  git-for-pairs
+| Create new local repo: `US_States`<br>Add `all_states.txt` file<br>Add title to `all_states.txt` file<br>Add, Commit, Push to GitHub | |
 | Add collaborator in GitHub | |
-| Checkout new branch: `north_carolina`<br>Add `north_carolina.txt` file<br>Add "North Carolina" to `all_states.txt` file<br>`git add` , `git commit` , `git push`
-
+| Checkout new branch: `north_carolina`<br>Add `north_carolina.txt` file<br>Add "North Carolina" to `all_states.txt` file<br>Add, Commit, Push to GitHub
 
 ### Adding collaborators
 
 We've added a new step to the process: adding a collaborator. Player 1 has invited Player 2 to contribute to the repo. There are several options when adding a collaborator; in this case, we want both developers to have equal access and rights to the repository. 
 
-> 4. P2: Accept GitHub invitation
-> 1. P2: Create new branch
+> 4. P2: [Accept GitHub invitation](https://www.loom.com/share/f0e4dce877054420877fcd51ddfad6ea)
 
 | Player 1 | Player 2 |
 | -------- | -------- |
 | | Accept invitation on GitHub<br>Clone `US_States` repo to local
-| | Checkout new branch: `colorado`<br>Add `colorado.txt` file<br>Add "Colorado" to `all_states.txt` file<br>`git add` , `git commit` , `git push`
+| | Checkout new branch: `colorado`<br>Add `colorado.txt` file<br>Add "Colorado" to `all_states.txt` file<br>Add, Commit, Push to GitHub
 
 ### Cloning a repository
 
-Player 2 has performed two important tasks. First, they have cloned the repo created by Player 1. To clone a repository means to copy a remote Github repository to your local computer using the following command:
-
-```
-git clone <SSH KEY for repository>
-```
-
-When you clone a repository, you interact with that repository by pushing and pulling branches. Therefore, when you are working with teammates, i.e. in a group project, you all need to clone the same repository so that you are all working in the same code base.
+Player 2 has performed two important tasks. First, they have cloned the repo created by Player 1. To clone a repository means to copy a remote Github repository to your local computer. You interact with that repository by pushing and pulling branches. Therefore, when you are working with teammates, i.e. in a group project, you all need to clone the same repository so that you are all working in the same code base.
 
 ### Pull Request
 
 Second, Player 2 has created content in a separate branch, but it not yet merged with the `main` branch. We've discussed how to merge branches, but we want one additional step to ensure that both developers agree on the content.
 
 To that end, Player 2 will create a Pull Request (PR). A **Pull Request** informs all collaborators of the changes made to a branch and the desire to merge this branch with another branch (usually the `main` branch).
 
-> 6. P2: Create Pull Request on GitHub
+> 5. P2: Create Pull Request on GitHub
```

Comment:
Is there a video for this step?

Reply:
there's not a separate video for creating a PR, it was in the Create New Branch video.